views/js/qtiCreator/widgets/static/math/states/Active.js

```diff
@@ -199,12 +199,12 @@ define([
             $panels.mathml.show();
             if($fields.latex.val()){
                 //show a warning here, stating that the content in LaTeX will be removed
-                if(!$fields.mathml.hasClass('tooltipstered')){
+                if(!$fields.mathml.data('qtip')){
                     _createWarningTooltip($fields.mathml);
                 }
-                $fields.mathml.tooltipster('show');
+                $fields.mathml.qtip('show');
                 $editMode.off('change.editMode').one('change.editMode', function(){
-                    $fields.mathml.tooltipster('hide');
+                    $fields.mathml.qtip('hide');
```

Comment:
Even if it was not done previously, you should add the module in the required dependencies, like you did for the imageSelector (https://github.com/oat-sa/extension-tao-itemqti/pull/482/files#diff-e36980090f6ee6e36bba144a2d00b188R28)